app/javascript/packs/public.jsx

```diff
@@ -1,22 +1,31 @@
 import './public-path';
-import escapeTextContentForBrowser from 'escape-html';
 import loadPolyfills from '../mastodon/load_polyfills';
-import ready from '../mastodon/ready';
 import { start } from '../mastodon/common';
+
+import escapeTextContentForBrowser from 'escape-html';
+import ready from '../mastodon/ready';
 import loadKeyboardExtensions from '../mastodon/load_keyboard_extensions';
 import 'cocoon-js-vanilla';
 import axios from 'axios';
 import { throttle } from 'lodash';
 import { defineMessages } from 'react-intl';
+import * as IntlMessageFormat  from 'intl-messageformat';
+import { timeAgoString }  from '../mastodon/components/relative_timestamp';
+import { delegate }  from '@rails/ujs';
+import * as emojify  from '../mastodon/features/emoji/emoji';
```

Comment:
This does not work. This causes `Uncaught (in promise) TypeError: emoji is not a function`. Reverting to `import emojify from '../mastodon/features/emoji/emoji` works.